tasks/utils.py

```diff
@@ -68,6 +68,47 @@ def get_multi_python_location(embedded_path=None, rtloader_root=None):
     return rtloader_lib, rtloader_headers, rtloader_common_headers
 
 
+def get_nikos_linker_flags(nikos_libs_path):
+    nikos_libs = [
+        'dnf',
+        'gio-2.0',
+        'modulemd',
+        'gobject-2.0',
+        'ffi',
+        'yaml',
+        'gmodule-2.0',
+        'repo',
+        'glib-2.0',
+        'pcre',
+        'z',
+        'solvext',
+        'rpm',
+        'rpmio',
+        'bz2',
+        'solv',
+        'gpgme',
+        'assuan',
+        'gcrypt',
+        'gpg-error',
+        'sqlite3',
+        'curl',
+        'nghttp2',
+        'ssl',
+        'crypto',
+        'json-c',
+        'lzma',
+        'xml2',
+        'popt',
+        'zstd',
```

Comment:
nit: Would be nice if these were sorted in some fashion

Reply:
The ordering of these flags is deliberate. If library A uses symbols which are defined in library B, then the flag for library B must appear after library A in order for the symbols to be resolved.